### PR TITLE
Bug 1716622: start informers after lister registration

### PR DESCRIPTION
**Pull request by sanchezl**
Picking up https://github.com/openshift/library-go/pull/435
To fix https://bugzilla.redhat.com/show_bug.cgi?id=1716622

### CODE REVIEW

**Comment by sanchezl** [comment]
/hold

fakebump until https://github.com/openshift/library-go/pull/435 merges

File: glide.yaml [hunk 1]
```diff
@@ -23,7 +23,8 @@ import:
 - package: github.com/openshift/client-go
   version: master
 - package: github.com/openshift/library-go
-  version: master
```
Comment (deads2k) [inline glide.yaml]: hold until this can just be release-4.1

File: glide.yaml [hunk 2]
```diff
@@ -23,7 +23,8 @@ import:
 - package: github.com/openshift/client-go
   version: master
```
Comment (deads2k) [inline glide.yaml]: make this release-4.1

Comment (sanchezl) [reply glide.yaml]: done
also pinned api

**Pull request merged**